### PR TITLE
Automated PR for CrystalStacker

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/CrystalStacker.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<SPDX name="CrystalStacker License" identifier="CrystalStacker" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing:CrystalStacker?rd=Licensing/CrystalStacker</url>
+  </urls>
+  <license>
+    <title>
+      <p>Crystal Stacker is freeware. This means you can pass copies around freely provided you include this
```
Comment (sam-ellis) [inline src/CrystalStacker.xml]: Lines 7-9 do not look like a title. Consider merging with optional or body.

Comment (jlovejoy) [inline src/CrystalStacker.xml]: agreed that it is not a title. I'd also say that we can simply remove the optional tag altogether

Comment (pmadick) [inline src/CrystalStacker.xml]: Agreed.  Suggest that this be considered part of the body of the license.

Comment (bradleeedmondson) [inline src/CrystalStacker.xml]: agreed, make part of license and remove &lt;optional&gt; tag around disclaimer

Comment (jlovejoy) [inline src/CrystalStacker.xml]: discussed on 8/16 legal call - remove title and optional tag

File: src/CrystalStacker.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<SPDX name="CrystalStacker License" identifier="CrystalStacker" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing:CrystalStacker?rd=Licensing/CrystalStacker</url>
+  </urls>
+  <license>
+    <p>Crystal Stacker is freeware. This means you can pass copies around freely provided you include this
```
Comment (sam-ellis) [inline src/CrystalStacker.xml]: Personally I would put all of the text within body tags, as none of it specifically looks like is it title or optional. Pull request submitted.

**Pull request merged**